mozregression/main.py

```diff
@@ -286,6 +287,24 @@ def preference(prefs_files, prefs_args):
     return prefs()
 
 
+def check_mozregression_version(logger):
+    url = "https://pypi.python.org/pypi/mozregression/json"
+    try:
+        mozregression_version = requests.get(url).json()['info']['version']
+    except (RequestException, KeyError, ValueError):
+        logger.critical("Unable to get latest version from pypi.")
+        return
+
+    if __version__ != mozregression_version:
+        logger.warn("You are using mozregression version %s, "
+                    "however version %s is available.",
+                    __version__, mozregression_version)
+
+        logger.warn("You should consider upgrading via the 'pip install"
+                    " --upgrade mozregression'\n or via"
```

Comment:
I would prefer something more simple here. Could you remove the part "\n or via..." ? It is quite redundant.